packages/ocean-react/src/Modal/Modal.tsx

```diff
@@ -12,6 +12,7 @@ export type ModalProps = {
   blocked?: boolean;
   /**
    * Determine the max-width of the modal (desktop only).
+   * If the maxWidth is set to 'lg', the modal will become fullscreen on smaller screens.
```

Comment:
`maxWidth` but affects `height`?

Reply:
@leonardoalemax How can property make more understandable without breaking change?

Reply:
`maxSize` maybe?

Reply:
wouldn't `size` be understandable enough in this scenario?

Reply:
`maxSize` + `blocked` = `size`

Reply:
@edu2711 @leonardoalemax escape route:
- `maxWidth`: Determine the max-width of the modal (desktop only)
- `maxHeight`: Determine the max-height of the modal (mobile only)
- `blocked`: If `true`, the modal stretches to `maxWidth` or `maxHeight`.

Reply:
What do you think? :point_up:

Reply:
Or `blocked` in mobile represents full screen always and we can leave maxWidth as is.

Reply:
I have had some problems with this prop too, i think we should not make user to choose from a list of sizes, we should pick one as a default, and give the possibility to user to set it the way it wanted overwriting the css of by prop. 

this is a front end library not a north korea hair cut.  


buuuuuuuuuuuuuut. 

we should also make the user life easier, sooo. i guess  

size [string]  "fullscreen" || "lg" ||  "md" ||  "sm"

if the someone want anything different than that, he could overwrite the css. 


also

mobile is always full screen, if not is an alert not a modal.

Reply:
I agree with @leonardoalemax .

`size [string] "fullscreen" || "lg" || "md" || "sm"`

I guess this approach is easier to understand.

"if the someone want anything different than that, he could overwrite the css."

Reply:
I agree with @leonardoalemax too.
We should proceed with this approach.

@edu2711 what is your position?